app/views/posts/index.html.erb

```diff
@@ -1 +1,24 @@
-<h1>Welcome to my Posts page!</h1>
\ No newline at end of file
+<div class="user">
+      <img src=<%=@user.photo%> alt="User image" height="150" width="150" >
+      <div class="user_details">
+        <h3><%= @user.name %></h3>
+        <p>Number of posts: <%=@user.post_counter%>
+      </div>
+</div>
```

Comment:
* After adding some data to the database by using the `rails console` [as instructed here](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md#use-models-to-insert-data) i was unable to see any post, as it is a requirement to make the views pages function as shown in the wireframes [See requirement here](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=Implement%20the%20design,everything%20functions%20first.). Kindly work on this 🙏